### PR TITLE
Login link related changes

### PR DESCRIPTION
**Pull request by fusionproj**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,11 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.6.3",
+    "bootstrap": "^4.6.0",
```
Comment (fusionproj) [inline package.json:9]: remove this

File: package.json [hunk 2]
```diff
@@ -6,9 +6,11 @@
...
     "git": "^0.1.5",
     "npm-git-install": "^0.3.0",
     "react": "^17.0.1",
+    "react-bootstrap": "^1.4.3",
```
Comment (fusionproj) [inline package.json:13]: remove this

File: src/Components/Navbar.js [hunk 3]
```diff
@@ -1,17 +0,0 @@
...
-               <ul>
-                   <li><Link to='/'>AboutUs</Link></li>
-                   <li><Link to='/Contact'>ContactUs</Link></li>
-               </ul>
-           </nav>
-       </BrowserRouter>
-       );
-};
-
-
```
Comment (fusionproj) [inline src/Components/Navbar.js:16]: Why we are delting this FILE

File: src/Components/Routes.js [hunk 4]
```diff
@@ -3,16 +3,25 @@ import React from 'react';
...
         <BrowserRouter>
+        <Navbar />
             <Switch>
-                <Route exact path='/' component={About} />
-                <Route exact path='/Contact' component={Contact} />
+                 <Route exact path='/Home' component={Home} /> 
+                <Route exact path='/About' component={About} /> 
+                <Route exact path='/Contact' component={Contact} /> 
+                <Route path='/' component={Login} />
+               <Route exact path='/Login' component={Login} />
```
Comment (fusionproj) [inline src/Components/Routes.js:23]: Why we are creating login as a new route and component

**Comment by fusionproj** [comment]
Please resolve conflicts from these files
Steps

Always pull changes from the main branch
Resolve conflict 
push your changes 
Raise a new PR with the latest changes

File: package.json (hunk 1 above)
Comment (ektabindal08) [reply package.json:9]: done

Comment (ektabindal08) [reply package.json:9]: done

**Pull request closed without merging**